src/app/docs/tabs/tabs-docs.component.html

```diff
@@ -4,10 +4,373 @@
+        
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
+      heading="Layout"
+    >
+        
+        <sky-docs-design-guideline>
+        <p>
+          Tabs are always the last (bottommost) content in their container. Don’t put additional content under tabs.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't put additional content under tabs."
+          captionType="danger"                                     
+          imageSource="~/assets/img/guidelines/tabs/tabs-layout-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+        
+      <sky-docs-design-guideline>
+        <p>
+          The tab border extends the full width of the container. Don’t use left or right margins on the tab border, and ignore any padding that the container defines.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do extend the tab border to the full width of the container."
+          captionType="success"                                     
+          imageSource="~/assets/img/guidelines/tabs/tabs-layout-2.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't apply the container's padding to the tab border."
+          captionType="danger"                                     
+          imageSource="~/assets/img/guidelines/tabs/tabs-layout-3.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+        
+      <sky-docs-design-guideline>
+        <p>
+          Tab content does not have additional left or right padding. The left and right gutters align with the gutters for content outside the tabs.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't apply additional left or right padding to tab content."
+          captionType="danger"                                     
+          imageSource="~/assets/img/guidelines/tabs/tabs-layout-4.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+        
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
+      heading="Related information"
+    >
+      <sky-docs-design-guideline
+        heading="Components"
+      >
+        
+          <ul>
+              <li><a href="https://developer.blackbaud.com/skyux/components/vertical-tabs">Vertical tabs</a></li>
+          </ul>
+          
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Guidelines"
+      >
+        
+          <ul>
+              <li><a href="https://developer.blackbaud.com/skyux/design/guidelines/content-containers">Content containers</a></li>
+              <li><a href="https://developer.blackbaud.com/skyux/design/guidelines/page-layouts">Page design</a></li>
+          </ul>
+          
+      </sky-docs-design-guideline>
+
```

Comment:
I'm not sure where to place this comment, so I'm sorry if this isn't the best place. ... In the Installation section on the Development tab, I just noticed that the code snippets after *Defined in* and *Installation* don't use the same styling as the code snippets on the rest of the page. It seems like they should. @Blackbaud-SteveBrush, is there a reason to format these samples differently? Or should I file an issue in the docs-tools repo?

Reply:
I noticed this difference a while back, but @Blackbaud-AdamFunderburk said it was intentional.

Reply:
I assume this is OK to disregard for now? We can ignore it for this PR and always update the styles separately. Does that sound like the best approach @Blackbaud-AdamFunderburk?

Reply:
I can recall the conversation, but honestly can't remember why we decided to leave these particular fields without the code snippet styling. IIRC we were testing it around the same time as we tried the new sky-code style in general, and maybe that other style looked out-of-place here? I don't think it's going to cause any trouble one way or the other.

Reply:
If you don't have a strong preference, I think we should update the code snippets in the Installation section to be consistent with the rest of the docs. But I don't think it should hold up this PR. @Blackbaud-SteveBrush, should I create an issue for that in the docs-tools repo?

Reply:
Yes, that would be helpful @blackbaud-johnly.

Reply:
Created the issue at https://github.com/blackbaud/skyux-docs-tools/issues/22.